<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-08-04T13:52:10 CEST" name="Suite" finished-at="2023-08-04T13:52:34 CEST" duration-ms="23500">
    <groups>
    </groups>
    <test started-at="2023-08-04T13:52:10 CEST" name="Test" finished-at="2023-08-04T13:52:31 CEST" duration-ms="20505">
      <class name="uk.co.automationtesting.AddRemoveItemBasketTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:uk.co.automationtesting.AddRemoveItemBasketTest@20ccf40b]" started-at="2023-08-04T13:52:11 CEST" name="setup" finished-at="2023-08-04T13:52:19 CEST" duration-ms="8242" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="addRemoveItem()[pri:0, instance:uk.co.automationtesting.AddRemoveItemBasketTest@20ccf40b]" started-at="2023-08-04T13:52:19 CEST" name="addRemoveItem" finished-at="2023-08-04T13:52:30 CEST" duration-ms="10696" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Cart amount did not match the expected amount, it found$45.24 expected $45.23]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Cart amount did not match the expected amount, it found$45.24 expected $45.23
at org.testng.Assert.fail(Assert.java:99)
at uk.co.automationtesting.AddRemoveItemBasketTest.addRemoveItem(AddRemoveItemBasketTest.java:81)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addRemoveItem -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:uk.co.automationtesting.AddRemoveItemBasketTest@20ccf40b]" started-at="2023-08-04T13:52:30 CEST" name="tearDown" finished-at="2023-08-04T13:52:31 CEST" duration-ms="760" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- uk.co.automationtesting.AddRemoveItemBasketTest -->
    </test> <!-- Test -->
    <test started-at="2023-08-04T13:52:10 CEST" name="Test2" finished-at="2023-08-04T13:52:34 CEST" duration-ms="23500">
      <class name="uk.co.automationtesting.OrderCompleteTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:uk.co.automationtesting.OrderCompleteTest@21282ed8]" started-at="2023-08-04T13:52:11 CEST" name="setup" finished-at="2023-08-04T13:52:19 CEST" duration-ms="8395" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="endToEndTest()[pri:0, instance:uk.co.automationtesting.OrderCompleteTest@21282ed8]" started-at="2023-08-04T13:52:19 CEST" name="endToEndTest" finished-at="2023-08-04T13:52:33 CEST" duration-ms="14206" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endToEndTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:uk.co.automationtesting.OrderCompleteTest@21282ed8]" started-at="2023-08-04T13:52:33 CEST" name="tearDown" finished-at="2023-08-04T13:52:34 CEST" duration-ms="388" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- uk.co.automationtesting.OrderCompleteTest -->
    </test> <!-- Test2 -->
  </suite> <!-- Suite -->
</testng-results>
